src/xenia/gpu/dxbc_shader_translator.cc

```diff
@@ -435,12 +435,19 @@ void DxbcShaderTranslator::StartVertexOrDomainShader() {
                                                   : dxbc::Dest::R(0, 0b0111),
                  dxbc::Src::VDomain(0b000110));
         if (register_count() >= 2) {
-          // Copy the primitive index to r1.x as a float.
+          // Copy the primitive index + offset to r1.x as a float.
```

Comment:
I can't remember if we already have a comment in the API-agnostic part of `xe::gpu`, but if yes, the need for the offset (referencing this title ID specifically and that it's used in ground draws there) should be documented there.